agent/main.go

```diff
@@ -252,6 +252,7 @@ func hostApi(socketHarOutputChannel chan<- *tapApi.OutputChannelItem) {
 	routes.EntriesRoutes(app)
 	routes.MetadataRoutes(app)
 	routes.StatusRoutes(app)
+	routes.ServiceMapRoutes(app)
```

Comment:
the routes should exposed only if the feature flag is enabled

Reply:
The front-end call the status endpoint to hide/show the service map button, so we must register the routes to be able check the status